.github/workflows/pkgdown.yaml

```diff
@@ -45,7 +53,37 @@ jobs:
         run: R CMD INSTALL .
 
       - name: Deploy package
+        if: contains(env.isPush, 'true')
         run: |
           git config --local user.email "actions@github.com"
           git config --local user.name "GitHub Actions"
           Rscript -e 'pkgdown::deploy_to_branch(new_process = FALSE)'
+
+      - name: Create website and another page
+        if: contains(env.isPush, 'false')
```

Comment:
Can you please use the run + shell technique? (See the install dependencies step above)

Reply:
done